src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,102 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final int DEFAULT_THRESHOLD = 12;
```

Comment:
let's use 0.75 in our logic
in fact, 0.75 is a constant, and threshold can be calculated on demand in code